packages/client/hmi-client/src/components/model/tera-model.vue

```diff
@@ -43,6 +43,7 @@
 			:highlight="highlight"
 			@update-model="updateModelContent"
 			@fetch-model="fetchModel"
+			:key="model?.id"
```

Comment:
needed to add a key here to properly refresh this component when navigating to and from different models